02-oop-inheritance/javaScript/refactored_JS/downloableMovie v0.1.js

```diff
@@ -0,0 +1,94 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function ( title ) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function ( paramName, paramValue ) {
+  //Variables
+  this.attributes = [];
+  this.attributes[paramName] = paramValue; 
+  this.observer = new MovieObserver();
+};//end movie constructor
+
+Movie.prototype = {
+  
```

Comment:
![pizarra 04](https://cloud.githubusercontent.com/assets/6402577/10017823/0427a46c-6108-11e5-8391-e5a75d7a0e13.jpg)

please, review my next commit. I left only the object properties in the constructor, as you told in the blackboard. 
The only attribute i left in the prototype was the 'new MovieObserver'.

Is that what you meant ? Please, specify if you don't